__tests__/app.test.js

```diff
@@ -51,6 +51,7 @@ describe("GET api/articles/:article_id", () => {
           "created_at",
           "votes",
           "article_img_url",
+          "comment_count"
```

Comment:
We should write an extra test not refactor the old one with the idea that the old test does not break if we change the behaviour of the endpoint. If it does than we change it, but it shows the importance of robust tests 🙂